packages/gestalt/src/Upsell.js

```diff
@@ -27,6 +29,10 @@ type Props = {|
     accessibilityLabel: string,
     onDismiss: () => void,
   |},
+  imageData?: {|
+    component: typeof Image | typeof Icon,
```

Comment:
can we do here:

`import React, { type Element, type Node } from 'react';
`

`    component: Element<typeof Image | typeof Icon>,
`